Sources/Sentry/SentrySDK.m

```diff
@@ -164,6 +164,10 @@ + (void)setStartTimestamp:(NSDate *)value
 
 + (void)startWithOptions:(SentryOptions *)options
 {
+#if SENTRY_TARGET_PROFILING_SUPPORTED
+    stopLaunchProfile();
```

Comment:
Hmm, yeah that is an interesting use case. They'd have to be selectively calling startWithOptions because they'd have had to call it on a prior launch to enable the launch profile in the first place. This is a tricky one to handle because although now we have a profile timeout, it will go away with continuous profiling.

Reply:
https://github.com/getsentry/sentry-cocoa/issues/3637